dedicated - cut all APIv6 /me/installationTemplate dependencies

JayBeeDe opened the pull request:
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | **MANAGER-15833:** <br /><br />- MANAGER-15902<br />- MANAGER-16046<br />- MANAGER-15929<br />- MANAGER-15927<br />- MANAGER-15876<br />- MANAGER-15873
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)

## Description

Cut all APIv6 /me/installationTemplate dependencies - MANAGER-15833

## Related

MANAGER-15833 is splited in the following sub-tasks:
- :new: MANAGER-15900: use dynamic rootmountpoint info rather than canbedelete
- :new: MANAGER-15902: add a way to identify whether lvm or not
- :worm: MANAGER-16046: HW raid: raid level sorting, default raid level and disks values, wrong max number of disks
- :new: MANAGER-15929: store HW raid & partitioning infos to scope
- :new: MANAGER-15927: stop creating tmp-mgr templates in os installation
- :worm: MANAGER-15876: only query hw raid details when server has raid hw
- :no_entry_sign: MANAGER-15873: remove server-installation-gabarit component

:warning: The following 2API middleware retro-compatible (=non-blocking) changes are required in order to work:
- :new: MANAGER-15900: Add partitioning scheme names list and lvmReady flag to sws-dedicated-server-installation-ovhtemplate
- :new: MANAGER-15901: sws-dedicated-server-installationTemplate-partitions should also handle ovh template

darsene commented on packages/manager/modules/bm-server-components/src/general-information/general-information.controller.js:
```diff
@@ -16,15 +16,21 @@ export default class BmServerComponentsGeneralInformationController {
   }
 
-  getOperatingSystemLabel() {
+  hasOperatinSystem() {
```
```suggestion
  hasOperatingSystem() {
```

darsene commented on packages/manager/modules/bm-server-components/src/general-information/general-information.controller.js:
```diff
@@ -16,15 +16,21 @@ export default class BmServerComponentsGeneralInformationController {
 
-    return os;
+  getOperatingSystemLabel() {
+    if (this.hasOperatinSystem()) {
```
```suggestion
    if (this.hasOperatingSystem()) {
```

darsene commented on packages/manager/modules/bm-server-components/src/general-information/general-information.html:
```diff
@@ -107,16 +107,12 @@
                 <span
-                    data-translate="dedicated_server_dashboard_installation_ovh_button"
+                    data-translate="dedicated_server_dashboard_installation_install_button"
+                    data-ng-if="!$ctrl.hasOperatinSystem()"
```
```suggestion
                    data-ng-if="!$ctrl.hasOperatingSystem()"
```

darsene commented on packages/manager/modules/bm-server-components/src/general-information/general-information.html:
```diff
@@ -107,16 +107,12 @@
                 <span
-                    data-translate="dedicated_server_dashboard_installation_gabarit_button"
+                    data-translate="dedicated_server_dashboard_installation_reinstall_button"
+                    data-ng-if="$ctrl.hasOperatinSystem()"
```
```suggestion
                    data-ng-if="$ctrl.hasOperatingSystem()"
```

JayBeeDe commented:
FYI I have just rebased from master, fixed conflicts in translation commit and also removed 1 commit that was created during the staging link creation with @sachinrameshn
